hackathon_site/dashboard/frontend/src/components/dashboard/TeamCard/TeamCard.tsx

```diff
@@ -2,24 +2,39 @@ import React from "react";
 import styles from "./TeamCard.module.scss";
 import Container from "@material-ui/core/Container";
 import Typography from "@material-ui/core/Typography";
-import Button from "@material-ui/core/Button";
 import TitledPaper from "components/general/TitledPaper/TitledPaper";
+import Button from "@material-ui/core/Button";
+import { useSelector } from "react-redux";
+import { teamCodeSelector, teamMemberNamesSelector } from "slices/event/teamSlice";
 
-const TeamCard = ({ members, teamCode, handleEditTeam }) => {
-    const title = teamCode === "" ? "Team" : "Team " + teamCode;
+interface TeamProps {
+    handleEditTeam?: any;
+}
 
+export const TeamCard = ({ handleEditTeam }: TeamProps) => {
+    const members = useSelector(teamMemberNamesSelector);
+    const teamCode = useSelector(teamCodeSelector);
+
+    const title = teamCode === null ? "Team" : "Team " + teamCode;
```

Comment:
Can we change this to if you don't have a team it should say "No Team"
And add a test for this (for when teamCode is null)